runtime/j9vm/j9vmnatives.xml

```diff
@@ -58,7 +58,6 @@
 		<export name="_JVM_GC@0" />
 		<export name="_JVM_GCNoCompact@0" />
 		<export name="_JVM_GetAllThreads@8" />
-		<export name="_JVM_GetCallerClass@8" />
```

Comment:
Please leave a comment to explain the export for _JVM_GetCallerClass is found in the module.xml files.

Reply:
Added comments, and updated copyrights as well.
Pls have another look.